Disables tests to avoid hang on stable/9 and head.

Temporary fix until correct solution is found.


Index: examples/Makefile.in
@@ -124,7 +124,7 @@
 #	$(DOXYGEN)/bin/doxygen dbusxml.cfg
 
 diagrams/html/index.html: diagrams_a.h diagrams_b.h diagrams_c.h diagrams_d.h diagrams_e.h diagrams.cfg
-ifneq ($(HAVE_DOT),)
-	$(DOXYGEN)/bin/doxygen diagrams.cfg
-endif
+#ifneq ($(HAVE_DOT),)
+#	$(DOXYGEN)/bin/doxygen diagrams.cfg
+#endif
